tasks/extract_filepaths_for_granule/API.md

```diff
@@ -118,25 +118,28 @@ Lambda handler. Extracts the key's for a granule from an input dict.
   other dictionary keys may be included, but are not used.
   other dictionary keys may be included, but are not used.
   
-- `Example` - {
-                "event": {
-                  "granules": [
-                    {
-                      "granuleId": "granxyz",
-                      "recoveryBucketOverride": "test-recovery-bucket",
-                      "version": "006",
-                      "files": [
-                        {
-                          "fileName": "file1",
-                          "key": "key1",
-                          "source": "s3://dr-test-sandbox-protected/file1",
-                          "type": "metadata"
-                        }
-                      ]
-                    }
-                  ]
-                }
-              }
+
+**Example**:
+
+  {
+- `"event"` - {
+- `"granules"` - [
+  {
+- `"granuleId"` - "granxyz",
+- `"recoveryBucketOverride"` - "test-recovery-bucket",
+- `"version"` - "006",
+- `"files"` - [
+  {
+- `"fileName"` - "file1",
+- `"key"` - "key1",
+- `"source"` - "s3://dr-test-sandbox-protected/file1",
+- `"type"` - "metadata"
+  }
+  ]
+  }
+  ]
+  }
+  }
```

Comment:
I am guessing this is created by build_api.sh which can't capture the original formatting of this json.
Should we manually update this json or keep it in this format?

Reply:
Leave it as-is. Otherwise, too many manual steps for something that's supposed to be automatic.